backend/models/core/Resource.js

```diff
@@ -0,0 +1,99 @@
+import DB from "@backend/data_access/DB";
+
+class Resource {
```

Comment:
Now that I thought about this, I think:

* this model should be named as a `Setting`
* the table should be named as `settings`
* the api should be callable as `/api/settings/mainlogo`